src/pages/navbar.js

```diff
@@ -1,14 +1,18 @@
 import React from 'react';
 import './Style.css';
+import { Icon } from '@iconify/react';
 
 const Navbar = () => (
   <div className="navbar">
     <div className="navbar-title">
-      <h1>Bookstore CMS</h1>
+      <h1 className="title-nav">Bookstore CMS</h1>
     </div>
     <div className="navbar-links">
-      <a className="links" href="/">Books</a>
-      <a className="links" href="/categories">Categories</a>
+      <a className="links-books" href="/">BOOKS</a>
+      <a className="links-categories" href="/categories">CATEGORIES</a>
+    </div>
+    <div className="circle-person">
+      <Icon className="icon" icon="bi:person-fill" />
     </div>
```

Comment:
- [Optional] Please align this icon with the main container as seen in the screenshots below:

Your Implementation | Zeplin Design
:---:|:---:
![](https://user-images.githubusercontent.com/63087491/167042657-2036802f-bc89-4fa2-b710-87ff6d7ed99c.png)| ![](https://user-images.githubusercontent.com/63087491/167042637-7d08c511-aaa6-430c-b89d-9a5f36e3d13f.png)

Reply:
Done :100: